osf/email/__init__.py

```diff
@@ -0,0 +1,308 @@
+import os
+import re
+import json
+import logging
+import importlib
+import sys
+from html import unescape
+from typing import List, Optional
+from mako.template import Template as MakoTemplate
+
+
+import waffle
+from django.core.mail import EmailMessage, get_connection
+
+from mako.lookup import TemplateLookup
+
+from sendgrid import SendGridAPIClient
+from python_http_client.exceptions import (
+    BadRequestsError as SGBadRequestsError,
+    HTTPError as SGHTTPError,
+    UnauthorizedError as SGUnauthorizedError,
+    ForbiddenError as SGForbiddenError,
+)
+
+from osf import features
+from website import settings
+
+def _existing_dirs(paths: List[str]) -> List[str]:
```

Comment:
Nit: as of python 3.10 we don't have to use `typing.List`, builtin list works the same way

```suggestion
def _existing_dirs(paths: list[str]) -> list[str]:
```